credentials/oauth_token.go

```diff
@@ -0,0 +1,8 @@
+package credentials
+
+type OAuthToken struct {
```

Comment:
For those less familiar with OAuth, let's add links to the RFC & document each field